CHANGELOG.md

```diff
@@ -445,7 +456,10 @@ See the [Tendermint v0.34.7 SDK changelog](https://github.com/tendermint/tenderm
 
 <!-- Release links -->
 
-[Unreleased]: https://github.com/cosmos/gaia/compare/v7.0.3...HEAD
+[Unreleased]: https://github.com/cosmos/gaia/compare/v8.0.0...HEAD
+[v8.0.0]: https://github.com/cosmos/gaia/releases/tag/v8.0.0
```

Comment:
Should we have the RC link, until the real release gets made?

Reply:
was never done before, i think it should be expecting the next actual release. The point is to create compare links so people can see the difference? https://github.com/olivierlacan/keep-a-changelog/issues/71

Reply:
Ok lets keep as is.